src/System.Linq.Expressions/tests/System.Linq.Expressions.Tests.csproj

```diff
@@ -23,6 +23,9 @@
     <Compile Include="$(CommonTestPath)\System\TypeBuilderExtensions.cs" />
   </ItemGroup>
   <ItemGroup>
+      <CodeCoverageAssemblies Include="Microsoft.CSharp" /> 
```

Comment:
You're correct. The extra green I'd seen locally when I tried this was just from the extra test introduced with the actual fix, not from bringing in the S.L.E tests on classes derived from those here. I've reverted back to `CodeCoverageAssemblies`.